src/Casdoor.Client/Extensions/QueryMapBuilder.cs

```diff
@@ -1,6 +1,6 @@
 namespace Casdoor.Client;
 
-public class QueryMapBuilder
+internal class QueryMapBuilder
```

Comment:
This class always be used by temporary variables, struct is better.

Reply:
done